runtime/doc/terminal.txt

```diff
@@ -144,8 +144,8 @@ directory indicated in the request. >lua
   vim.api.nvim_create_autocmd({ 'TermRequest' }, {
     desc = 'Handles OSC 7 dir change requests',
     callback = function(ev)
-      if string.sub(vim.v.termrequest, 1, 4) == '\x1b]7;' then
-        local dir = string.gsub(vim.v.termrequest, '\x1b]7;file://[^/]*', '')
+      if string.sub(ev.data.payload, 1, 4) == '\x1b]7;' then
+        local dir = string.gsub(ev.data.payload, '\x1b]7;file://[^/]*', '')
```

Comment:
`ev.data.data` is probably the right name, though it's a bit redundant I guess. Not a blocker.